### DISCUSSION TITLE
Support for query parameters in OpenAPI export/import

### DISCUSSION THREAD

**Discussion by jicornelius** [mockoon/mockoon#439] [Ideas] [answered]
Will there be support for query parameters in the OpenAPI import and export?  If you import an OpenAPI document into Mockoon and then export it out to OpenAPI json, the only parameters that are included are those in the path.  A large portion of the OpenAPI document is lost.  Since you cannot author full OpenAPI specifications in Mockoon you have to do that in a separate tool and if you add additional methods etc, there is no easy way import those changes into an existing mock environment.

**Comment by 255kb** [comment, marked as answer]
Hi @jicornelius thank you for your message. 
Indeed, Mockoon is not an API design tool. Some tools like Stoplight are better at this. 
We focus on mocking, and do not handle anything related to the request. 
While I understand it could be great to have a 1:1 equivalent between Mockoon and OpenAPI specifications it's not on the roadmap.

**Reply by jicornelius** [reply]
Thanks for the response.
